worker/peergrouper/worker.go

```diff
@@ -591,7 +593,7 @@ func (w *pgWorker) updateReplicaSet() (map[string]*replicaset.Member, error) {
 	if err := setHasVote(added, true); err != nil {
 		return nil, errors.Annotate(err, "adding new voters")
 	}
-	if desired.isChanged {
+	if w.config.SupportsHA && desired.isChanged {
```

Comment:
It's probably good to have a comment to explain why we disable this for Non HA controller.

Reply:
ok